cpp/test/CMakeLists.txt

```diff
@@ -116,7 +116,7 @@ if(BUILD_PRIMS_TESTS)
     # (please keep the filenames in alphabetical order)
     add_executable(prims
       prims/add.cu
-      prims/add_sub_dev_scalar.cu
+      prims/adjustedRandIndex.cu
```

Comment:
why have you removed the `add_sub_dev_scalar.cu` from the tests?

Reply:
oh my bad!